File: 04.py
class Solution:
    def findMedianSortedArrays(self, nums1: list[int], nums2: list[int]) -> float:
        for i in nums2:
            nums1.append(i)
        z = sorted(nums1)
        valueMid = len(z)//2
        if len(z) % 2 == 0:
            f = (z[valueMid-1]+z[valueMid])/2
            return f
        else:
            f1 = z[valueMid]
            return f1

File: 04.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function (nums1, nums2) {
  nums2.forEach((num) => {
    nums1.push(num);
  });
  nums1.sort((a, b) => a - b);
  if (nums1.length % 2 === 0) {
    return (nums1[nums1.length / 2 - 1] + nums1[nums1.length / 2]) / 2;
  } else {
    return nums1[(nums1.length - 1) / 2];
  }
};
